.github/workflows/test_with_drupal.yaml

```diff
@@ -0,0 +1,37 @@
+name: Test With Drupal
+
+on:
+    pull_request: null
+    push:
+        branches:
+            - master
+
+jobs:
+    test_with_drupal:
+        runs-on: ubuntu-latest
+        steps:
+            -   uses: actions/checkout@v2
+            -   uses: shivammathur/setup-php@v1
+                with:
+                    php-version: 7.3
+                    coverage: none
+
+            -   name: Install Drupal project
+                run: |
+                    # bring rector dependencies
+                    composer install --no-progress
+
+                    # Install latest Drupal version
+                    composer create-project drupal/recommended-project drupal
+                    cd drupal
```

Comment:
The path issues might be caused by this line.

`cd ..` is only applied per `run:`, so next `run:` is again in root directory

Reply:
Better have just one long `run:` with all the commands  than think about all the location issues.